90zfsbootmenu/zfsbootmenu-init.sh

```diff
@@ -93,6 +94,16 @@ while true; do
     continue
   fi
 
+  if [ "${zbm_import_attempt}" -lt "${zbm_import_retries:-0}" ]; then
+    # The max number of import attempts has not been reached;
+    # retry unless the user decides to abort immediately
+    zbm_import_attempt="$((zbm_import_attempt + 1))"
+    if delay="${zbm_import_delay:-5}" prompt="Unable to import pool, retrying in %0.2d seconds" \
+        timed_prompt "[RETURN] to retry immediately" "[ESCAPE] for a recovery shell"; then
+      continue
```

Comment:
Can you throw in a zdebug or zinfo statement here indicating that it's on import attempt X of Y?